app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,7 @@
+.name {
+  font-size: 150%;
+}
+
+#in_panel {
```

Comment:
Avoid using id selectors<br>Selector `in_panel` should be written in lowercase with hyphens